styles/pages/internship.scss

```diff
@@ -0,0 +1,88 @@
+.testimonial-item {
+  box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.3), 0 0.0625rem 0.125rem rgba(0, 0, 0, 0.2);
+  margin: 3px;
+  padding: 5px;
+}
+
+p {
+  font-size: 0.8em;
+  line-height: 150%;
+  margin: 0 22px 14px;
+
+  &.subheading {
+    font-size: 1em;
+  }
+}
+
+.application-information {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.application-item {
+  margin-bottom: 15px;
+  margin-top: 10px;
+  text-align: left;
+
+  p {
+    margin-left: 15%;
+    margin-top: 5px;
+  }
+}
+
+.application-info-card {
+  background-color: #FFF;
+  border-radius: 1rem;
+  box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.3), 0 0.0625rem 0.125rem rgba(0, 0, 0, 0.2);
+  font-size: 1.3em;
+  margin: auto;
+  max-width: 250px;
+  padding: 20px;
+  padding-top: 0;
+  text-align: left;
+}
+
+.application-button {
+  text-align: center;
+  width: 100%;  
+}
+
+.testimonial-section {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  margin-top: 20px;
+}
+
+.testimonial-giver {
+  text-align: right;
+}
+
+.faq {
+  .faq-row-wrapper {
+    .faq-body {
+      text-align: left;
```

Comment:
Could we just use `.text-left` or something similar here?

Reply:
can't! it's a component from a package, so I have to use their class hierarchy to edit any styles they don't let us edit directly using `FAQStyles`

Reply:
Oh I see. In that case, let's add a comment that explains this, and links to the relevant component - since that component might change in the future, it would be good to know what we'd need to update.